File: src/question/month08/day0827/gebal_0827.c
#include<stdio.h>

int main() {
	int cache[4][3] = { {3,5,9},{2,11,5},{8,30,10},{22,5,1} }, sum = 0;
	for (int i = 0; i < 4; i++) {
		for (int j = 0; j < 3; j++) {
			sum += cache[i][j];
			printf("%d ", cache[i][j]);
		}
		printf("\n");
	}
	printf("%d", sum);
}

File: src/question/month08/day0827/gebal_0827.py
import numpy as np
cache = np.array([[3, 5, 9], [2, 11, 5], [8, 30, 10], [22, 5, 1]])
[[print(v, end=' ') if i != 2 else print(v) for i, v in enumerate(x)] for x in cache]
print(cache.sum())
